### ISSUE TITLE
[arm64] Test faile: JIT_SIMD._BitwiseOperations_ro_BitwiseOperations_ro_._BitwiseOperations_ro_BitwiseOperations_ro_sh

### ISSUE THREAD

**Issue by VincentBu** [dotnet/coreclr#23803]
**Job:**
https://ci.dot.net/job/dotnet_coreclr/job/master/job/arm64_cross_debug_ubuntu16.04_tst/159/

**Failed tests:**
JIT_SIMD._BitwiseOperations_ro_BitwiseOperations_ro_._BitwiseOperations_ro_BitwiseOperations_ro_sh

**Log:**
```
JIT_SIMD._BitwiseOperations_ro_BitwiseOperations_ro_._BitwiseOperations_ro_BitwiseOperations_ro_sh [FAIL]
        /n/nReturn code:      1/nRaw output file:      /home/robox/j/workspace/dotnet_coreclr/master/arm64_cross_debug_ubuntu16.04_tst/bin/tests/Linux.arm64.Debug/JIT/SIMD/Reports/JIT.SIMD/BitwiseOperations_ro/BitwiseOperations_ro.output.txt/nRaw output:/nBEGIN EXECUTION
        /home/robox/j/workspace/dotnet_coreclr/master/arm64_cross_debug_ubuntu16.04_tst/bin/tests/Linux.arm64.Debug/Tests/Core_Root/corerun BitwiseOperations_ro.exe ''
        Expected: 100
        Actual: 0
        END EXECUTION - FAILED
        Test Harness Exitcode is : 1
        /nTo run the test:/n> set CORE_ROOT=/home/robox/j/workspace/dotnet_coreclr/master/arm64_cross_debug_ubuntu16.04_tst/bin/tests/Linux.arm64.Debug/Tests/Core_Root/n> /home/robox/j/workspace/dotnet_coreclr/master/arm64_cross_debug_ubuntu16.04_tst/bin/tests/Linux.arm64.Debug/JIT/SIMD/BitwiseOperations_ro/BitwiseOperations_ro.sh/n
        Expected: True
        Actual:   False
        Stack Trace:
          /mnt/j/workspace/dotnet_coreclr/master/arm64_cross_debug_ubuntu16.04/bin/tests/Linux.arm64.Debug/TestWrappers/JIT.SIMD/JIT.SIMD.XUnitWrapper.cs(599,0): at JIT_SIMD._BitwiseOperations_ro_BitwiseOperations_ro_._BitwiseOperations_ro_BitwiseOperations_ro_sh()
```

**Comment by AndyAyersMS** [comment]
Assuming codegen for now.

cc @dotnet/jit-contrib

**Comment by RussKeldorph** [comment]
@dotnet/jit-contrib Is anyone besides @CarolEidt available to take this?

**Comment by sandreenko** [comment]
```
robox@ubunddarm64s-016:~/sandreenko/git/coreclr/bin/tests/Linux.arm64.Checked/JIT/SIMD/BitwiseOperations_ro$ ./BitwiseOperations_ro.sh -coreroot=/home/robox/sandreenko/git/coreclr/bin/tests/Linux.arm64.Debug/Tests/Core_Root
BEGIN EXECUTION
/home/robox/sandreenko/git/coreclr/bin/tests/Linux.arm64.Debug/Tests/Core_Root/corerun BitwiseOperations_ro.exe ''
Expected: 100
Actual: 0
END EXECUTION - FAILED
robox@ubunddarm64s-016:~/sandreenko/git/coreclr/bin/tests/Linux.arm64.Checked/JIT/SIMD/BitwiseOperations_ro$ ./BitwiseOperations_ro.sh -coreroot=/home/robox/sandreenko/git/coreclr/bin/tests/Linux.arm64.Checked/Tests/Core_Root
BEGIN EXECUTION
/home/robox/sandreenko/git/coreclr/bin/tests/Linux.arm64.Checked/Tests/Core_Root/corerun BitwiseOperations_ro.exe ''
Expected: 100
Actual: 100
END EXECUTION - PASSED
```

First build with the issue: 
https://ci.dot.net/job/dotnet_coreclr/job/master/job/arm64_cross_debug_ubuntu16.04_tst/158/

**Comment by sandreenko** [comment]
Confirmed that this issue was also caused by https://github.com/dotnet/coreclr/pull/23344. I will check if it was fixed by #23889 or not.

**Comment by sandreenko** [comment]
Was fixed by  #23889.

**Comment by CarolEidt** [comment]
Thanks @sandreenko for following up on this!
